packages/wonder-blocks-form/src/components/__tests__/text-field.test.tsx

```diff
@@ -740,4 +746,901 @@ describe("TextField", () => {
         const input = await screen.findByRole("textbox");
         expect(input).toHaveAttribute("aria-invalid", "false");
     });
+
+    describe("Validation", () => {
```

Comment:
These validation tests are the same as the validation tests added for TextArea in [PR#2344](https://github.com/Khan/wonder-blocks/pull/2355/files#diff-0b9f5a7a93ed66290b63ac82420db318d5dafd407741b5013def25790bedffd7), but renamed to work with TextField.

I was thinking about whether we needed these tests + the 1TextArea1 tests + the 1useFieldValidation1 tests. I think it doesn't hurt to verify all of these places, especially since there are lots of edge cases. One difference is `TextField` and `TextArea` will test for the `aria-invalid` attribute, while the `useFieldValidation` tests check the `errorMessage`

 Let me know though if anyone has thoughts on this!

Reply:
Good question. I agree that increasing coverage is good in these cases. I feel that we are really trying to cover edge cases (not just following a coverage number), so keeping these tests make sense.

As for the `TextField` + `TextArea` tests, we have a `consistency-tests` folder that we use when we want to test similar behaviors for common components. I wonder if it would make sense to move these tests to that folder of if it's definitely better to keep them with the other tests. how do you feel about this? (tbh it's totally fine if we keep them separated)

cc @marcysutton in case you have opinions as well.

Reference: https://github.com/Khan/wonder-blocks/tree/main/consistency-tests/__tests__

Reply:
Oh cool, I didn't know we had consistency tests in WB!

I think for this case, since this validation logic is specific to these 2 components, we can keep them separated. If we were to have a third component with the same validation logic, then I think we could move the common tests there!

Reply:
All great points! and agree on SearchField as well.